backend/typescript/services/implementations/camperService.ts

```diff
@@ -308,6 +308,48 @@ class CamperService implements ICamperService {
     }
   }
 
+  async getCampersByChargeIdAndSessionId(
+    chargeId: string,
+    sessionId: string,
+  ): Promise<CamperDTO[]> {
+    try {
+      // eslint-disable-next-line prettier/prettier
+      const campers: Camper[] = await MgCamper.find({ chargeId, campSession: sessionId });
+
+      if (!campers || campers.length === 0) {
+        throw new Error(
+          `Campers with Charge Id ${chargeId} and Session Id ${sessionId} not found.`,
+        );
+      }
```

Comment:
That's a good point, I only added this since the function above (`getCampersByChargeId`) also throws this same error in its implementation. I think we should stay consistent with that, so we should either remove both throws or keep both, but I'm unsure if the throw in `getCampersByChargeId` has any significance (it's in the code so it might), so I rather not delete that... thoughts?